runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -489,7 +489,7 @@ getCurrentThread(J9UpcallMetaData *data, bool *isCurThrdAllocated)
 }
 
 /**
- * @brief Converts the type of the return value to the return type intended for JEP389/419 upcall.
+ * @brief Converts the type of the return value to the return type intended for upcall.
```

Comment:
Please add a  comment saying VMAccess is required, and caller needs to do exceptoin check after